Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/InternetExplorer/InternetExplorerSpecificBrowserConfigurationExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using Coypu.Drivers;
+using JetBrains.Annotations;
+using Remotion.Utilities;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure.InternetExplorer
+{
+  public static class InternetExplorerSpecificBrowserConfigurationExtensions
+  {
+    /// <summary>
+    /// Gets a flag indicating if <paramref name="browserConfiguration"/> represents <b>Internet Explorer</b> by testing if the passed object
+    /// implements <see cref="IInternetExplorerConfiguration"/>.
+    /// </summary>
+    public static bool IsInternetExplorer ([NotNull] this IBrowserConfiguration browserConfiguration)
+    {
+      ArgumentUtility.CheckNotNull ("browserConfiguration", browserConfiguration);
+
+      return browserConfiguration is IInternetExplorerConfiguration;

```

Comment:
change from itnerface to class, the itnerface goes bye-bye.

probably the extensions type will get renamed when both extension methods are moved here.